plugins/coverbrowser.koplugin/mosaicmenu.lua

```diff
@@ -648,6 +661,30 @@ function MosaicMenuItem:update()
                 self.has_description = true
             end
 
+            if not self.menu.cover_info_cache then
+                self.menu.cover_info_cache = {}
+            end
+
+            local percent_finished, status
+            if bookinfo then
+                if self.menu.cover_info_cache[self.filepath] then
+                    percent_finished, status = unpack(self.menu.cover_info_cache[self.filepath])
+                else
+                    local docinfo = DocSettings:open(self.filepath)
+                    -- We can get nb of page in the new 'doc_pages' setting, or from the old 'stats.page'
+                    if docinfo.data.summary and docinfo.data.summary.status then
+                        status = docinfo.data.summary.status
+                    end
+                    percent_finished = docinfo.data.percent_finished
+                    self.menu.cover_info_cache[self.filepath] = {percent_finished, status}
+                end
+                if status == "complete" then
+                    -- Display these instead of the read %
+                    percent_finished = nil
```

Comment:
Just making sure you really want this: no progress bar if book mark as completed ?
(Update that comment if that's the case.)

Reply:
Let's show instead if it is complete.